### PR TITLE
Created sphinx docs based on current README

### PR DESCRIPTION
**Pull request by rubiconjosh**
I created sphinx docs based on the information in the current README.

The docs use Sphinx which I did not add to requirements.txt as I am unsure if you want to make it optional.

I did not setup an account on Read The Docs as I assume you want control of that.

I do want to contribute significantly more to the documentation, but I want to make sure I am on the right track first.

#27

### CODE REVIEW

**Comment by overshard** [comment]
This is great and exactly what I want to do! I have a few issues before I merge this and I'll detail those out later tonight like the author and copyright I'd rather be Timestrap's Contributors and not just me since I would like to be community inclusive. I'd also like to discuss Markdown vs RST. I personally prefer Markdown but I know the python community, and pypi, use RST so that may be what we end up using even though it isn't my preference.

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'Timestrap'
+copyright = '2017, overshard'
```
Comment (overshard) [inline docs/conf.py]: Change copyright to 2017, Timestrap's Contributors

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+author = 'overshard'
```
Comment (overshard) [inline docs/conf.py]: Same change to Author, Timestrap's Contributors

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'Timestrap', 'Timestrap Documentation',
+     author, 'Timestrap', 'One line description of project.',
```
Comment (overshard) [inline docs/conf.py]: One line description should be `Time tracking and invoicing you can host anywhere. Full export support in multiple formats and easily extensible.`

**Comment by overshard** [comment]
Oh, one more thing! If you can add the recommonmark and Sphinx requirements to the development requirements file in `requirements/development.py`.

**Comment by overshard** [comment]
I've went ahead and setup this project on Read The Docs as of now: http://timestrap.readthedocs.io/en/latest/ ... nothing there yet till this pull though.

**Comment by rubiconjosh** [comment]
I have made the changes requested. 

The organization is a little sloppy right now. I captured all of the information from the current readme.md except for testing.

I want to the docs to be organized as follows if you agree:

Installation
------------
 - General
 - Linux
 - Windows

Deployment
-------------
 - Heroku
 - PythonAnywhere

Development
--------------
 - Overview
 - Testing

Also for development.txt I was unsure of what versioning to put for the packages.

**Comment by overshard** [comment]
This works for me, I'm fine with that organization. Great start thank you!

**Pull request merged**